external-crates/move/move-compiler/src/parser/ast.rs

```diff
@@ -1272,13 +1281,20 @@ impl AstDebug for StructDefinition {
 
         w.write(&format!("struct {}", name));
         type_parameters.ast_debug(w);
-        if let StructFields::Defined(fields) = fields {
-            w.block(|w| {
+        match fields {
+            StructFields::Defined(fields) => w.block(|w| {
                 w.semicolon(fields, |w, (f, st)| {
                     w.write(&format!("{}: ", f));
                     st.ast_debug(w);
                 });
-            })
+            }),
+            StructFields::Positional(types) => w.block(|w| {
+                w.semicolon(types.iter().enumerate(), |w, (i, st)| {
+                    w.write(&format!("pos{}: ", i));
+                    st.ast_debug(w);
+                });
+            }),
+            StructFields::Native(_) => unreachable!(),
```

Comment:
This shouldn't be unreachable?

Reply:
Ah good point -- should just do nothing. Will fix